core/chains/evm/gas/arbitrum_estimator.go

```diff
@@ -133,6 +134,19 @@ func (a *arbitrumEstimator) GetLegacyGas(ctx context.Context, calldata []byte, l
 	return
 }
 
+// During network congestion Arbitrum's suggested gas price can be extremely volatile, making gas estimations less accurate. For any transaction, Arbitrum will only charge
+// the block's base fee. If the base fee increases rapidly there is a chance the suggested gas price will fall under that value, resulting in a fee too low error.
+// We use gasPriceWithBuffer to increase the estimated gas price by some percentage to avoid fee too low errors. Eventually, only the base fee will be paid, regardless of the price.
+func (a *arbitrumEstimator) gasPriceWithBuffer(gasPrice *assets.Wei, maxGasPriceWei *assets.Wei) *assets.Wei {
+	gasPrice = gasPrice.AddPercentage(25)
```

Comment:
@prashantkumar1982 we could potentially make the 25% value configurable, although I'm a bit worried about having an ENV var that is specifically dedicated to 1 chain.

Reply:
I think its ok not to have it be configurable, but can it be at least in a named constant?